app/samples/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    package="com.sygic.samples">
+
+    <application
+        android:allowBackup="false"
+        android:fullBackupContent="false"
+        android:icon="@mipmap/ic_launcher"
+        android:label="@string/app_name"
+        android:roundIcon="@mipmap/ic_launcher_round"
+        android:supportsRtl="true"
+        android:theme="@style/AppTheme"
+        tools:ignore="GoogleAppIndexingWarning">
+
+        <activity
+            android:name=".activities.SamplesActivity"
+            android:label="@string/app_name"
+            android:theme="@style/AppTheme.NoActionBar">
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN" />
+
+                <category android:name="android.intent.category.LAUNCHER" />
+            </intent-filter>
+        </activity>
+        <activity
+            android:name=".BrowseMapDefaultActivity"
+            android:label="@string/browse_map_default" />
+        <activity
+            android:name=".BrowseMapFullActivity"
+            android:label="@string/browse_map_full" />
+        <activity
+            android:name=".BrowseMapModesActivity"
+            android:label="@string/browse_map_modes" />
+        <activity
+            android:name=".BrowseMapClickListenerActivity"
+            android:label="@string/browse_map_click_listener" />
+        <activity
+            android:name=".BrowseMapMarkersActivity"
+            android:label="@string/browse_map_markers" />
+        <activity
+            android:name=".BrowseMapThemesActivity"
+            android:label="@string/browse_map_themes"
+            android:theme="@style/Theme.Sygic.Colored" />
```

Comment:
respective tu to mas aj v xml, aj z kodu sa to setne

Reply:
aha, tak tu jsem to zapomněl smazat, odstraním ;)